azkaban-exec-server/src/test/java/azkaban/execapp/FlowRunnerTest.java

```diff
@@ -69,9 +77,11 @@ public void setUp() throws Exception {
     jobtypeManager =
         new JobTypeManager(null, null, this.getClass().getClassLoader());
     JobTypePluginSet pluginSet = jobtypeManager.getJobTypePluginSet();
-    pluginSet.addPluginClass("java", JavaJob.class);
+    pluginSet.setCommonPluginLoadProps(AllJobExecutorTests.setUpCommonProps());
     pluginSet.addPluginClass("test", InteractiveTestJob.class);
     fakeProjectLoader = new MockProjectLoader(workingDir);
+    Utils.initServiceProvider();
+    AzkabanExecutorServer.setApp(new FakeApp());
```

Comment:
> How does FlowRunner depend on AzkabanExecutorServer?

Maybe the only need to have `AzkabanExecutorServer` initialized comes from this line in `FlowRunner`:

    logger.info("Assigned executor : " + AzkabanExecutorServer.getApp().getExecutorHostPort());

I guess this could be avoided quite easily at least.